src/pages/Dashboard/inputComponent.definitions.ts

```diff
@@ -0,0 +1,10 @@
+export interface ReusableInputProps {
+  name: string;
+  register: any;
+  errors: any;
+  isRequired: boolean;
+  label: string;
+  placeholder: string;
+  defaultValue: string;
+  errorText: string;
```

Comment:
Can you add the `type` property in here as well so that we can pass that to the underying input component. That should just be a string. We want this so we can pass in things like `text`, `email`, `password`, etc.